app-admin/pwman3/pwman3-0.12.2.ebuild

```diff
@@ -24,8 +25,13 @@ RDEPEND="
 "
 BDEPEND="test? ( dev-python/pexpect[${PYTHON_USEDEP}] )"
 
+
 distutils_enable_tests setup.py
 
+python_test() {
+	python -m tests.test_pwman
```

Comment:
This is missing a || die, and should be using ${EPYTHON}.

Reply:
fixed.